ci-operator/step-registry/aws/provision/edge-zone-subnet/aws-provision-edge-zone-subnet-ref.yaml

```diff
@@ -1,19 +1,19 @@
 ref:
-  as: aws-provision-localzone-subnet
+  as: aws-provision-edge-zone-subnet
   from_image:
     namespace: ocp
     name: "4.12"
     tag: upi-installer
   grace_period: 10m
-  commands: aws-provision-localzone-subnet-commands.sh
+  commands: aws-provision-edge-zone-subnet-commands.sh
   resources:
     requests:
       cpu: 10m
       memory: 100Mi
   env:
-  - name: EDGE_ZONE_TYPE
+  - name: EDGE_ZONE_TYPES
```

Comment:
I think `EDGE_ZONE_TYPE` is better than `EDGE_ZONE_TYPES`, `EDGE_ZONE_TYPES` make me feel like it is a multi-items value.

Reply:
Ignore it, there is some discussion in https://github.com/openshift/release/pull/47045#discussion_r1446297816